src/main/java/nl/tudelft/contextproject/model/Inventory.java

```diff
@@ -69,51 +75,56 @@ public void remove(Entity ent) {
 				}
 			}
 		}
-		return;
 	}
 	
 	/**
 	 * Returns a bomb from the inventory.
+	 *
 	 * @return
-	 * 		A bomb from the inventory
+	 * 		a bomb from the inventory if the inventory contains a bomb
```

Comment:
"if the inventory contains a bomb, returns that bomb. Otherwise returns null"
is probably better.